packages/cli-config/src/cli/action.import.ts

```diff
@@ -280,7 +280,18 @@ async function outputChange(
   }
 
   if (styleUpdate.length > 0) md.push('# Vector Style Update', ...styleUpdate);
-  if (vectorUpdate.length > 0) md.push('# Vector Data Update', ...vectorUpdate);
+  if (vectorUpdate.length > 0) {
+    md.push('# Vector Data Update', ...vectorUpdate);
+
+    const layerPath = (await mem.TileSet.get('ts_topographic-v2'))?.layers?.[0]?.[3857];
```

Comment:
Can we split change this in the PR and make a bug ticket for the problem? Thanks.